.github/workflows/ci-app-examples.yml

```
@@ -32,7 +32,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, macOS-11, windows-2022]
-        pkg-name: ["LAI", "app"]  # the "LAI" is for installing monolithic package
+        pkg-name: ["app"]  # TODO: add "LAI" # "LAI" installs the monolithic package
```

```suggestion
        pkg-name: ["app", "LAI"]  # "LAI" installs the monolithic package
```
this is already working

---

Let's see if it's true

---

Still failing: 

https://github.com/Lightning-AI/lightning/actions/runs/3313663726/jobs/5471949250